util/container/Dockerfile

```diff
@@ -2,151 +2,91 @@
 # Licensed under the Apache License, Version 2.0, see LICENSE for details.
 # SPDX-License-Identifier: Apache-2.0
 
-# Docker container for Snitch development.
+FROM ubuntu:22.04 AS builder
 
-# 1. Stage
-FROM ubuntu:18.04 AS builder
-ARG CMAKE_VERSION=3.19.4
-
-COPY apt-requirements.txt /tmp/apt-requirements.txt
-RUN apt-get update && \
-    sed 's/#.*//' /tmp/apt-requirements.txt \
-        | xargs apt-get install -y && \
+# apt update and upgrade
+RUN apt-get update && apt-get upgrade -y && \
     apt-get install -y \
         build-essential \
+        device-tree-compiler \
         curl \
         git \
         gnupg2 \
         lsb-release \
         software-properties-common \
+        tar \
         unzip \
         wget \
-        zlib1g-dev
+        zlib1g-dev \
+        zsh \
+        vim \
+        nano 
 
-# Build Rust tools
+# Install Rust
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-ENV PATH "/root/.cargo/bin:${PATH}"
+ENV PATH="/root/.cargo/bin:${PATH}"
 RUN rustup install 1.76.0
 RUN rustup override set 1.76.0
 
 # Install Bender
 RUN cargo install bender --version 0.27.1
```

Comment:
These are not the most recent versions. Just FYI
Rust is at 1.77.2 and bender is at 0.28.1

Reply:
I bumped the versions